examples/datadog-tracing/example.rst

```diff
@@ -0,0 +1,105 @@
+.. _install_sandboxes_datadog:
+
+Datadog tracing
+===============
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+The Datadog tracing sandbox demonstrates Envoy's :ref:`request tracing <arch_overview_tracing>`
+capabilities using `Datadog <https://datadoghq.com/>`_ as the tracing provider.
+
+This example includes a Datadog Agent which will forward to Datadog's backend traces sent from Envoy and the upstream `http` service.
+
+The ``envoy`` service is exposed on port ``10000`` and the request flow is as follow:
+
+    User -> ``envoy`` -> ``http``
+
+The Envoy proxy is configured (:download:`envoy.yaml <_include/datadog-tracing/envoy.yaml>`) to generate and propagate tracing context to upstream services and also to
+report tracing data to the Datadog Agent through an Envoy cluster nammed ``datadog_agent``.
+
+Each span records the latency of upstream API calls as well as information needed to correlate the span with other related spans (e.g., the trace ID).
+
+Step 1: Build the sandbox
+*************************
+
+Change directory to ``examples/datadog-tracing`` in the Envoy repository.
+
+To build this sandbox example, and start the example services run the following commands:
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/datadog-tracing
+    $ export DD_API_KEY=<YOUR_API_KEY>
+    $ docker compose pull
+    $ docker compose up --build -d
+    $ docker compose ps
+
+                Name                          Command             State            Ports
+    -----------------------------------------------------------------------------------------------------------
+    datadog-tracing-dd-agent-1   "/bin/entrypoint.sh"             running      8125/udp, 0.0.0.0:8126->8126/tcp
+    datadog-tracing-envoy-1      "/docker-entrypoint.…"           running      10000/tcp, 0.0.0.0:10000->1337/tcp
+    datadog-tracing-http-1       "node --require dd-t…"           running      0.0.0.0:8080->8080/tcp
+
+Step 2: Make a request to ``http``
+**********************************
+
+Now send a request to the ``http`` service, by calling http://localhost:10000.
```

Comment:
```suggestion
Now send a request to the HTTP service, by calling http://localhost:10000.
```

im aware there is a load of existing usage that makes http literal - but i think it should be unliteralized and capitals (as its an acronym)